habitat-hitl/habitat_hitl/core/remote_client_state.py

```diff
@@ -491,7 +495,7 @@ def get_new_connection_records(self) -> List[ConnectionRecord]:
 
     def on_frame_end(self) -> None:
         for user_index in self._users.indices(Mask.ALL):
-            self._gui_inputs[user_index].on_frame_end()
+            self._gui_inputs[user_index].reset(reset_continuous_input=False)
```

Comment:
Why is the `reset_continuous_input` hardcoded here? Move it to config or set at class init-time?

Reply:
I'm leaning towards avoiding to introduce an additional state to `GuiInput`.

The main upside of passing the parameter here is that it exposes the underlying complexity at the relevant line of code, as opposed to passing the argument via constructor.

I don't think that this should be driven by config either because there's no case where you would want to deviate from this implementation.